### ISSUE TITLE
Maybe a bug in layers.py (get_gaussian_weights)

### ISSUE THREAD

**Issue by Paranioar** [CrossmodalGroup/GSMN#23]
***def get_gaussian_weights(self, pseudo_coord):***
...
        weights = weights_rho * weights_theta                                        (**Line117**)
        weights[(weights != weights).detach()] = 0                                  (**Line118**)
                                                                                                              (**Line119**)                         
        # normalise weights                                                                      (**Line120**)
        weights = weights / torch.sum(weights, dim=1, keepdim=True) (**Line121**)
...

The dimension of weights should be  **(batch_size * K, neighbourhood_size, n_kernels)**, 
but the actual dimension is **(batch_size * K * neighbourhood_size, n_kernels)**, 
meaning that the weights are normalized on **n_kernels** instead of **neighbourhood_size** channels,
and it is different from the operations in **compute_weights(self, neighbourhood_weights)** (**Line227**).

I wonder if this is intentional or not, and I‘m not sure whether it affects network performance.
Maybe add 【**weights = weights.view(batch_size * K, neighbourhood_size, -1)**】in **Line119**.
